Description: fix a few spelling errors
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: no
Last-Update: 2014-01-28

--- a/rc.h
+++ b/rc.h
@@ -128,7 +128,7 @@
 	/* for the formatting routines */
 	va_list args;
 	long flags, f1, f2;
-	/* for the buffer maintainence routines */
+	/* for the buffer maintenance routines */
 	char *buf, *bufbegin, *bufend;
 	int flushed;
 	void (*grow)(Format *, size_t);
--- a/which.c
+++ b/which.c
@@ -1,7 +1,7 @@
 /* which.c: check to see if a file is executable.
 
    This function was originally written with Maarten Litmaath's which.c as
-   a template, but was changed in order to accomodate the possibility of
+   a template, but was changed in order to accommodate the possibility of
    rc's running setuid or the possibility of executing files not in the
    primary group. Much of this file has been re-vamped by Paul Haahr.
    I re-re-vamped the functions that Paul supplied to correct minor bugs
--- a/input.c
+++ b/input.c
@@ -229,7 +229,7 @@
 	save_lineno = istack->saved;
 }
 
-/* flush input characters upto newline. Used by scanerror() */
+/* flush input characters up to newline. Used by scanerror() */
 
 extern void flushu() {
 	int c;
--- a/glob.c
+++ b/glob.c
@@ -143,7 +143,7 @@
 }
 
 /*
-   lglob() globs a pattern agains a list of directory roots. e.g., (/tmp /usr /bin) "*"
+   lglob() globs a pattern against a list of directory roots. e.g., (/tmp /usr /bin) "*"
    will return a list with all the files in /tmp, /usr, and /bin. NULL on no match.
    slashcount indicates the number of slashes to stick between the directory and the
    matched name. e.g., for matching ////tmp/////foo*
--- a/lex.c
+++ b/lex.c
@@ -333,7 +333,7 @@
 }
 
 extern void scanerror(char *s) {
-	flushu(); /* flush upto newline */
+	flushu(); /* flush up to newline */
 	yyerror(s);
 	errset = prerror = TRUE;
 }
